Xamarin.Forms.Platform.Android/Renderers/BoxRenderer.cs

```diff
@@ -44,16 +48,76 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 
 			if (e.PropertyName == BoxView.ColorProperty.PropertyName || e.PropertyName == VisualElement.BackgroundColorProperty.PropertyName)
 				UpdateBackgroundColor();
+			else if (e.PropertyName == BoxView.CornerRadiusProperty.PropertyName)
+				UpdateCornerRadius();
 		}
 
 		protected override void UpdateBackgroundColor()
 		{
 			Color colorToSet = Element.Color;
 
-			if (colorToSet == Color.Default)
-				colorToSet = Element.BackgroundColor;
+			if (_backgroundDrawable == null)
```

Comment:
Are there any backwards compatibility implications now every box will have a gradientdrawable as a background?

Reply:
We use SetBackground method from ViewExtensions. In this extension method internally there are an API Level verification. Used in other Renderers like FrameRenderers.